apps/meteor/app/lib/server/startup/settings.ts

```diff
@@ -555,6 +555,18 @@ settingsRegistry.addGroup('Accounts', function () {
 			i18nLabel: 'Use_Legacy_Message_Template',
 			alert: 'This_is_a_deprecated_feature_alert',
 		});
+
+		this.add('Accounts_Default_User_Preferences_omnichannelTranscriptEmail', false, {
+			type: 'boolean',
+			public: true,
+			i18nLabel: 'Transcript_Email',
+		});
+
+		this.add('Accounts_Default_User_Preferences_omnichannelTranscriptPDF', false, {
+			type: 'boolean',
+			public: true,
+			i18nLabel: 'Transcript_PDF',
+		});
```

Comment:
Can you please move this to `apps/meteor/ee/app/livechat-enterprise/server/settings.ts`, so this setting gets created only when the license is present?